### DISCUSSION TITLE
magit(?) prompting "configure local settings"

### DISCUSSION THREAD

**Discussion by ville-h** [magit/magit#4947] [Support] [answered]
After a recent magit update when ever I try to launch `magit` I am prompted with a buffer: "configure local settings" and then bunch of things. Is this magit or is this git? If it's magit then why do I have to configure anything as its been working for years with out? How do I even configure them? How do I get: "do whatever you did before this started happening"?

**Comment by tarsius** [comment]
> Is this magit or is this git?

I don't know; that's a pretty vague description. Maybe a screenshot would help.

**Comment by ville-h** [comment]
![emacs_magit_configure_local_settings_000](https://github.com/magit/magit/assets/8824715/ef7741d8-238b-420b-9377-619767f17120)

**Comment by ville-h** [comment, marked as answer]
Mystery solved. It is my error: at some point I've managed to fat-finger the definition of the keyboard shortcut that is supposed to launch `magit` to launch `magit-notes`. Nevermind.
